client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -916,42 +916,45 @@ export default {
   },
 
   mounted () {
-    if (!this.$store.state.Place.loading && this.places.length === 0) {
-      this.fetchPlaces({
-        fields: {
-          Place: [
-            'description',
-            'name',
-            'solved',
-            'sortIndex'
-          ].join()
-        },
-        sort: 'sortIndex'
-      })
-        .then(() => {
-          this.setSelectedPlace()
-        })
-    } else {
-      this.setSelectedPlace()
-    }
-
-    if (!this.$store.state.AssignableUser.loading && this.assignableUsers.length === 0) {
-      this.fetchAssignableUsers({
-        fields: {
-          AssignableUser: [
-            'firstname',
-            'lastname'
-          ].join()
-        },
-        include: 'department',
-        sort: 'lastname'
-      })
-        .then(() => {
-          this.setSelectedAssignee()
-        })
-    } else {
-      this.setSelectedAssignee()
-    }
+    console.log('mounted segment')
```

Comment:
Should the changes in this file be committed?

Reply:
yes. This is the most important line of the PR ;-)

Reply:
this and some more leftovers where removed here 17f49e6c75b43d1271a3a511ba4a8ef72a5d754f